### PR TITLE
Support yml dist files

### PR DESCRIPTION
**Pull request by SimonBaeumer**
Support for .psh.yml.dist files and .psh.yaml.dist files

### CODE REVIEW

File: src/Config/YamlConfigFileLoader.php [hunk 1]
```diff
@@ -123,9 +138,7 @@ private function extractData(string $key, array $rawConfig, $default = false)
      */
     private function loadFileContents(string $file): string
     {
-        $contents = file_get_contents($file);
-
-        return $contents;
+        return file_get_contents($file);
```
Comment (JanPietrzyk) [inline src/Config/YamlConfigFileLoader.php:141]: :+1:

File: tests/Integration/Config/ConfigFileFinderTest.php [hunk 2]
```diff
@@ -11,7 +11,7 @@ public function test_config_loader_throw_when_it_cant_find_a_psh_file()
     {
         $loader = new ConfigFileFinder();
 
-        $this->setExpectedException(\RuntimeException::class);
+        $this->expectException(\RuntimeException::class);
```
Comment (JanPietrzyk) [inline tests/Integration/Config/ConfigFileFinderTest.php:14]: :+1:

**Review by JanPietrzyk — APPROVED** [review]
können wa machen :)

File: src/Config/YamlConfigFileLoader.php (hunk 1 above)
Comment (bcremer) [reply src/Config/YamlConfigFileLoader.php:141]: Any reason to keep that private method that is just a proxy to file_get_contents with another name?

**Pull request merged**